control-plane-operator/main.go

```diff
@@ -64,6 +64,9 @@ const (
 	// TODO: Include konnectivity image in release payload
 	konnectivityServerImage = "registry.ci.openshift.org/hypershift/apiserver-network-proxy:latest"
 	konnectivityAgentImage  = "registry.ci.openshift.org/hypershift/apiserver-network-proxy:latest"
+
+	// Default AWS KMS provider image. Can be overriden with annotation on HostedCluster
+	awsKMSProviderImage = "registry.ci.openshift.org/hypershift/aws-encryption-provider:latest"
```

Comment:
I don't see how it would be possible to change this image through an annotation, what am I missing?

Reply:
https://github.com/openshift/hypershift/blob/main/control-plane-operator/controllers/hostedcontrolplane/kas/params.go#L285-L287